subsystem/intake.py

```diff
@@ -0,0 +1,55 @@
+import config
+import constants
+from robotpy_toolkit_7407.motors import TalonConfig, TalonSRX
+from robotpy_toolkit_7407 import Subsystem
+
+
+PID = TalonConfig(1, 0, 1, 0)
+
+class Intake(Subsystem):
+
+    roller_motor: TalonSRX = TalonSRX(config.roller_id, config=PID)
+    hinge_motor: TalonSRX = TalonSRX(config.hinge_id, config=PID)
+    deployed: bool = False
+    target: float
+    tolerance: float = 0.02
+
+    def __init__(self):
+        super().__init__()
+        self.deployed = False
+
+    def init(self):
+        self.roller_motor.init()
+        self.hinge_motor.init()
+        self.hinge_motor.set_sensor_position(constants.intake_init_pos)
+
+    def deploy(self):
+        self.target = constants.intake_deploy_pos
+        self.hinge_motor.set_target_position(self.target)
```

Comment:
set_target_position() takes radians as input. Convert self.target (which is in rotations) into radians and also include the gear ratio (ask CAD ppl)